src/vm/jitinterface.cpp

```diff
@@ -6732,23 +6732,21 @@ DWORD CEEInfo::getMethodAttribsInternal (CORINFO_METHOD_HANDLE ftn)
         result |= CORINFO_FLG_NOSECURITYWRAP;
     }
 
+    if (IsMdRequireSecObject(attribs))
+    {
+        // Assume all methods marked as DynamicSecurity are
```

Comment:
Can you please also add a similar comment at https://github.com/dotnet/coreclr/pull/9491/files#diff-6291818ea5a59bda1055bcd3e1d3dcddL12 so that it becomes obvious at the definition about its semantics?